quick_start.adoc

```diff
@@ -26,6 +26,42 @@ We will first walk through setting up your managed cluster set and placement, th
 
 TODO Chris Ahl please add details here, including bringing clusters under management of a hub.
 
+[#creating-auth-realm]
+== Creating your AuthRealm
+
+The AuthRealm custom resoure specifies a combination of one or more https://docs.openshift.com/container-platform/4.8/authentication/understanding-identity-provider.html[identity providers] and a placement that determines to which managed clusters the identity provider(s) should be applied.
+
+An AuthRealm resource example is as follows:
+[source,yaml]
+----
+apiVersion: identityconfig.identitatem.io/v1alpha1
+kind: AuthRealm
+metadata:
+  name: identity-config-developers
+  namespace: identity-config-authrealm
+spec:
+  identityProviders:
+  - github:
+      clientID: 41c30d83fd1b4774f871
+      clientSecret:
+        name: identitatem-github-client-secret
+      organizations:
+      - identitatem
+    mappingMethod: claim
```

Comment:
I would change this from claim to add

Reply:
`claim` is the default value and is more restrictive than `add`. I feel like if someone is going to choose `add` they should do so thoughtfully. @tpouyer thoughts?

Reply:
I'm going to agree with @robinbobbitt here.  `claim` is the default in OCP and `add` brings with it certain "security concerns" that we don't want to "default" a customer to accepting.

Reply:
Ok, works for me.  But you may want to add a troubleshooting topic for claim/add and user